pandas/tests/arithmetic/test_array_ops.py

```diff
@@ -34,3 +35,26 @@ def test_object_comparison_2d():
     right.flags.writeable = False
     result = comparison_op(left, right, operator.ne)
     tm.assert_numpy_array_equal(result, ~expected)
+
+
+@pytest.fixture(params=[Index, Series, tm.to_array])
+def box_pandas_1d_array(request):
+    """
+    Fixture to test behavior for Index, Series and tm.to_array classes
+    """
+    return request.param
+
+
+@pytest.mark.parametrize(
+    "data, expected", [([0, 1, 2], [0, 2, 4])],
+)
+def test_extension_array_add(box_pandas_1d_array, box_1d_array, data, expected):
```

Comment:
this is specific to IntegerArray right, not general EA?  can you make the name more specific

This file is intended to directly test ops.array_ops, so this test should either go in test_numeric.py or a new test_integer_na.py

Reply:
Renamed test to `test_integer_array_add_list_like` and moved it to `test_numeric.py`

Reply:
thanks